hangman_assignment.js

```diff
@@ -0,0 +1,58 @@
+document.addEventListener("DOMContentLoaded", () => {
```

Comment:
This app is semi-working with some major glitches to be fixed on the way. Here are few things that I noticed: 
1. Lives are going down by increment equal to word length. 
2. Currently, there's no winning message if the user wins, nor any message to indicate the user lost. 
3. letters that are already guessed still gets added to letters guessed array. 
Overall, it requires fixes, but I also see that you understand how to create and manipulate DOM elements based on needs. Keep workin on this project, and let me know if you need any help.